webapp/tests/components/integrations/installed_command.test.jsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import * as Utils from 'utils/utils.jsx';
+import InstalledCommand from 'components/integrations/components/installed_command.jsx';
+
+describe('components/integrations/InstalledCommand', () => {
+    const emptyFunction = jest.fn();
+    const command = {
+        id: Utils.generateId(),
```

Comment:
Generating an id is causing the test to fail, it needs to be a constant (can be anything) so the snapshot gets the same value every time